MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include MANIFEST.in README.md AUTHORS.txt LICENSE.txt CHANGELOG.md
 recursive-include notifications *.py *.html *.txt *.po
+recursive-include notifications/static *
```

Comment:
@verdan Is this change needed for the Django 5.1 fix? If yes, could you please describe why?

Reply:
I don't think this should be here.